05-allow-traffic-from-all-namespaces.md

```diff
@@ -18,7 +18,7 @@ non-whitelisted traffic to all pods in the namespace](03-deny-all-non-whiteliste
 Start a web service on `default` namespace:
 
 ```sh
-kubectl run --generator=run-pod/v1 web --image=nginx \
+kubectl run web --image=nginx \
     --namespace default \
     --labels=app=web --expose --port 80
```

Comment:
Replace --labels=app=web with --labels="app=web"

Reply:
Also --namespace default should become --namespace=default

Reply:
Done